### DISCUSSION TITLE
UDL color pallette to Sketch

### DISCUSSION THREAD

**Discussion by ksabniskrishna** [twilio-labs/paste#718] [Q&A] [answered]
Can I add the UDL color pallette as color presets in sketch? Not talking about the design tokens - but just need them as presets in my sketch to add to any element (especially on non paste elements). Or is there a better way to add the pallette?

**Comment by richbachman** [comment]
Hi @ksabniskrishna. I'm checking with the team, and will get back with you as soon as possible.

**Comment by kate-porter** [comment, marked as answer]
Hey @ksabniskrishna - what version of Sketch are you on? We've got the UDL ("Default") theme ready to use, but it's recommended for Sketch 67.2 and above. If you're on a version below that and have an expired license key, IT can help you get a new one, and then we can get you set up with the Default theme once you've updated.

**Reply by ksabniskrishna** [reply]
Thanks Katie! I have the new license now and I have the default theme - but seems like everytime I want to use the colors from the theme, I can do it only on a given symbol - can I not add colors to non paste symbols?

**Reply by kate-porter** [reply]
Reposting from our Slack convo: do you mind watching [this video](https://www.loom.com/share/7f3fe097ad9d4c5abf2a5cd0cdd0846a) and see if it answers your question? tl;dr, we’ve setup our themes to use layer styles, not color swatches, and that video will walk you through how to use layer styles to create custom compositions. The video uses the Console theme as an example, but it's the same process to use the Default theme.
